source/win-main/DocumentTabs.vue

```diff
@@ -316,11 +316,19 @@ export default {
 <style lang="less">
 @tabbar-height: 30px;
 
+body {
+  &.dark {
+    div #tab-container {
+      border-top: 1px solid rgb(30, 30, 30);
```

Comment:
This can be merged into the section below by refactoring the `body div#tab-container` below to:

```less
body {
  div#tab-container {
    // some styles
  }

  &.dark div#tab-container {
    // some dark styles
  }
}
```